.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 exclude: '^(.*egg.info.*|.*/parameters.py|docs/).*$'
 default_language_version:
-    python: python3.8
+    python: python3.9
 repos:
 -   repo: https://github.com/asottile/pyupgrade
```

Comment:
We should eventually swap the `--py38-plus` flag to `--py39-plus`, but doing so introduces a lot of changes (mostly replacing type annotations like `List` with `list`). I'm unsure if this should be part of this PR or a separate one.

Reply:
Actually, given that we're still supporting Python 3.8, we should probably keep it as py38-plus.